buildpacks/go/bin/build

```diff
@@ -24,7 +24,7 @@ EOF
 cp $build_dir/bin/faas "$app_layer/faas"
 
 # REMOVE SOURCE CODE FROM FINAL CONTAINER
-rm -rf $build_dir/*
+# rm -rf $build_dir/*
```

Comment:
@lkingland is this useful? is it really in run image?

Reply:
I don't think this line is necessary.

This line was added on request from someone else (been a long time now; cant remember who). I was  pretty sure, when I put it together originally, that source was not included in the final run image.  But right now I can not remember exactly how the layers get assembled.  Sorry I can't remember more ATM!